apple/internal/apple_framework_import.bzl

```diff
@@ -111,6 +116,16 @@ def _apple_framework_import_impl(ctx):
         if filtered_framework_imports:
             transitive_sets.append(depset(filtered_framework_imports))
     else:
+        if ctx.attr.alwayslink:
+            framework_dir = framework_groups.keys()[0]
+            framework_name = paths.split_extension(paths.basename(framework_dir))[0]
+            framework_short_path = paths.join(framework_dir, framework_name)
+            framework_file = [
+                x
+                for x in framework_imports
+                if x.short_path == framework_short_path
+            ][0]
+            objc_provider_fields["force_load_library"] = depset([framework_file])
```

Comment:
It seems like there might be a better way to find this file. I also hoped we could just pass all the `framework_imports` but this list isn't filtered by type later so that doesn't work

Reply:
done